triaging.rst

```diff
@@ -505,13 +505,7 @@ with the "open" status.
 
 Mercurial Repository
 ''''''''''''''''''''
-HTTP link to a Mercurial repository that contains a patch for the issue.
-A :guilabel:`Create Patch` button will appear that computes a diff for the
-head revision of the remote branch and attaches it to the issue.  The button
-supports only CPython_ patches.
-
-If you don't indicate a remote branch, ``default`` is used.  You can
-indicate a remote branch by adding ``#BRANCH`` to the end of the URL.
+This field is not used anymore.
```

Comment:
```suggestion
Deprecated: HTTP link to a Mercurial repository that contains a patch for the issue.

New repository links should not be added to new or existing issues.
```

I also just opened a PR on the tracker to remove the ability to add new hg repo links: psf/bpo-tracker-cpython#8.  If that gets merged (and works :)), we could just remove this section.

Reply:
Great!